modules/ROOT/pages/sso-use-case-2.adoc

```diff
@@ -9,31 +11,36 @@ The instructions in this case are using Okta as an example identity provider how
 
 
 == Before You Begin
-Ensure you have the following setup:
+Ensure you have the following setup and context:
 
 * An identity provider for Anypoint Platform is already configured.
 * The user or the identity of the group of the users who have access to the application have been created in the identity provider. 
+* Read the information in the xref:enabling-sso-for-your-portal.adoc#before-you-begin [Before You Begin] section to understand the roles and permissions required and the information that API Experience Hub expects to configure custom user attributes and claims for the identity provider.
```

Comment:
```suggestion
* Understand the required roles, permissions, and information that API Experience Hub expects to configure custom user attributes and claims for the identity provider.
+
See xref:enabling-sso-for-your-portal.adoc#before-you-begin [Before You Begin].
```

Reply:
(Maybe that changes the meaning....)

Is the "AEH expects" coming from the UI - or is this identity-provider terminology? It feels less than clear to me. I wonder if you could just say "requires" instead.